src/express/app.js

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import cookieMiddleware from "./middlewares/cookieMiddleware";
+import queryMiddleware from "./middlewares/queryMiddleware";
+import setCookieMiddleware from "./middlewares/setCookieMiddleware";
+import checkMiddleware from "./middlewares/checkMiddleware";
+import routes from "./routes";
+import bodyParser from "body-parser";
+const app = express();
+
+//for post routes
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(routes);
+
+app.use(setCookieMiddleware); //testing
```

Comment:
Interesting position for helper middlewares. They will be executed only in case if route is not found.